docs/picontrol.md

```diff
@@ -0,0 +1,77 @@
+  - As a corollary, "PiControl" is probably a bad name for this system as pretty much all of the logic is hardware-agnostic. Even the small Python script run on the Pi could probably run on a much simpler microcontroller (with a couple modifications) through Micropython.
+- The keyboard emulation NFC readers were used because Katz (the author of this doc) had never touched hardware in his life and thus couldn't figure out how to use the [an actual breakout board](https://www.adafruit.com/product/364). In the first week of CS107e, he realized that he was connecting the board's TX and RX pins to the Pi's TX and RX pins (respectively). He has now learned that TX == "transmit" and RX == "receive" and thus TX on one board should be connected to RX on the other and vice versa. If you ever see Katz, feel free to chastise him for his foolishness.
+
+## Software
+
+### On the Pi
+
+The script run on the Pi can be found at https://github.com/MYKatz/PiControl. If you're using the aforementioned USB NFC readers, you should be running 'read_keyboard.py'.
+
+The script is very simple: upon receiving input from the NFC reader (a card's ID), it sends that to a specified API endpoint.
+
+#### OS and dependencies
+
+We used a standard install of Raspbian. You will need to configure the Pi to connect to Wifi -- there are many guides online to do this, such as [this one](https://raspberrypihq.com/how-to-connect-your-raspberry-pi-to-wifi/).
+
+An installation of Python 3.7 should include all of the necessary dependencies except for [Requests](https://requests.readthedocs.io/en/master/). A 'pip3 install requests' should work, provided you're connected to the internet.
+
+### Web App
+
+The PiControl web app comprises the vast majority of the logic. You can find it [here](https://github.com/treehacks/picontrol).
+
+PiControl is developed on Postgres, Go (w/ Gin), and React. You can call this the PGRG stack if you hate pronounceable acronyms.
+
+When the script on a Pi is ran for the first time, the Pi is registered with Picontrol and will show up on the 'Pis' page. You can then name the Pi and associate it with an event. When badges are scanned onto the Pi, they'll appear in the logs.
+
+#### Eventive Integration
+
+PiControl works by scanning NFC IDs into events on Eventive, which does the heavy lifting matching these IDs to hackers. If you're going to use Eventive (and you probably should, it's great!) then remember to update the EVENTIVE_API_KEY environment variable and change the eventive API link in the code (which is currently hard-coded.. oops!)
+
+#### Environment Variables
+
+```
+DATABASE_URL= /* postgresql uri string */
+EVENTIVE_API_KEY=
+GIN_MODE=release /* either 'debug' or 'release' */
+PASSWORD=
+
+```
+
+#### Development Set-up
+
+- Set "homepage" in package.json to the url you want.
```

Comment:
What URL should this be set to? Why does it need to be set?

Reply:
Yeah, I was horrible in not specifying. This sets the root url of the frontend to be whatever "homepage" is, but I don't think that's necessary to include in the documentation, as it should be "/admin" in order for everything to work anyway. Will remove